felix/fv/infrastructure/topology.go

```diff
@@ -57,6 +57,7 @@ type TopologyOptions struct {
 	VXLANMode                 api.VXLANMode
 	WireguardEnabled          bool
 	WireguardEnabledV6        bool
+	WireguardThreadingEnabled bool
```

Comment:
@jrcichra one last comment. Do we need this? If seems we are not using this flag. Am I missing it?

Reply:
We do not need it. Removed. Thanks!